core/cas-server-core-services/src/test/java/org/apereo/cas/authentication/DefaultMultifactorTriggerSelectionStrategyTests.java

```diff
@@ -168,6 +169,6 @@ private static RegexRegisteredService mockPrincipalService(final String provider
     
     private Principal mockPrincipal(final String attrName, final String... attrValues) {
```

Comment:
Could this be moved to CoreAuthenticationTestUtils? I vaguely remember a method call in there that mocks principal creation?